metecho/api/serializers.py

```diff
@@ -175,7 +188,9 @@ class Meta:
     def create(self, validated_data):
         instance = super().create(validated_data)
         instance.create_gh_branch(self.context["request"].user)
-        instance.queue_available_task_org_config_names(self.context["request"].user)
+        instance.project.queue_available_org_config_names(
+            user=self.context["request"].user
+        )
```

Comment:
@davisagli I'm curious about this. I'm adding code that automatically refreshes the list of available org config names from the repo when the relevant modal is opened (either for add-task, or create-playground-org), but right now I have it doing that _only if there are no known org configs already_. But if we also remove this line, that means once the list is fetched initially for a project, there's no automatic way that it will get updated if new options are added to the repo -- only if a user manually clicks the "refresh" button.

We could either replace this with an automatic refresh _every time_ a user opens the modal (even if we already have a known list), or we could go the other way and just remove this line, meaning we would never automatically refresh and would instead rely on users clicking the "refresh" btn.

My leaning is to take a middle ground and leave this line in, so that org configs are updated periodically -- every time a new Epic is created for that Project.

Does that makes sense? I'm curious what approach you prefer.

Reply:
@jgerigmeyer Got it. Yeah, I think that makes sense.